tests/integration/api/test_activation.py

```diff
@@ -174,9 +173,7 @@ def test_create_activation_bad_entity(admin_client: APIClient):
     [True, False],
 )
 @pytest.mark.django_db
-@mock.patch("aap_eda.core.tasking.is_redis_failed", return_value=True)
 def test_create_activation_redis_unavailable(
```

Comment:
This test name is wrong, as we don't expect redis anymore. I see in the code several references to redis not addressed in the PR. Do you expect more PR's until complete the migration? It its the case, I recommend to keep clear in the PR descriptions, which right now says "complete migration..."
Otherwise this PR should be revisited for completeness with all the concerns that I already shared.

Reply:
I'll address all comments together in this PR